python/paddle/hapi/model.py

```diff
@@ -1706,33 +1706,26 @@ def eval_batch(
 
         Examples:
 
-            .. code-block:: python
+            .. code-block:: pycon
 
                 >>> import paddle
                 >>> import paddle.nn as nn
                 >>> from paddle.static import InputSpec
                 >>> paddle.seed(2023)
 
-                >>> device = paddle.set_device('cpu') # or 'gpu'
+                >>> device = paddle.set_device('cpu')  # or 'gpu'
 
-                >>> net = nn.Sequential(
-                ...     nn.Linear(784, 200),
-                ...     nn.Tanh(),
-                ...     nn.Linear(200, 10))
-                ...
+                >>> net = nn.Sequential(nn.Linear(784, 200), nn.Tanh(), nn.Linear(200, 10))
```

Comment:
```suggestion
                >>> net = nn.Sequential(nn.Linear(784, 200), nn.Tanh(), nn.Linear(200, 10),)
```

加上 magic trailing comma 后重新格式化